WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -204,6 +217,8 @@ extension HubMenuViewModel {
     }
 
     private enum Localization {
+        static let payments = NSLocalizedString("Payments",
+                                                comment: "Title of one of the hub menu options")
```

Comment:
Should we change this comment to be more descriptive for the translators? Something like:

```
static let payments = NSLocalizedString("Payments", comment: "Title of the Payments menu in the hub menu")
```

Reply:
Yeah! Done in 59baeebcfa38be9b6ee4d43b92cd91c0d9e19187